README.md

```diff
@@ -17,6 +17,16 @@ for F5 BIG-IP Controller for Kubernetes.
 For guides on this and other solutions for Kubernetes, see the
 [F5 Solution Guides for Kubernetes](http://clouddocs.f5.com/containers/latest/kubernetes).
 
+Getting Help
+------------
+
+We encourage you to use our [Slack channel](https://f5cloudsolutions.slack.com/messages/C7NJD8LGH) for discussion and assistance on this
```

Comment:
This link is fine for F5 employees since it redirects to a page with instructions on how to create an account if you are an F5 employee. You need a link to https://f5cloudsolutions.herokuapp.com/ for non F5 users to be able to join the f5CloudSolutions org in Slack, and then point them to the cc-kubernetes channel.

Reply:
sure, will update the link and text.

Reply:
updated, and re-sent.